### PR TITLE
first level assignments

### PR DESCRIPTION
**Pull request by MatthewRGonzalez**
I want to submit this initial PR so you have an idea of where I'm at. I'll make more specific comments on certain lines.

### CODE REVIEW

File: data-paper/presentation.rmd [hunk 1]
```diff
@@ -8,11 +8,12 @@ output:
     slide_level: 3
 header-includes:
   - \setbeamertemplate{navigation symbols}{}
-  - \AtBeginDocument{\title[Quantifying U.S. GATT Trade Liberalization  \hspace{2.95in}\insertframenumber/\inserttotalframenumber]{Quantifying U.S. GATT Trade Liberalization}}
+  - \AtBeginDocument{\title[Quantifying U.S. GATT Trade Liberalization  \hspace{2.60in}\insertframenumber/\inserttotalframenumber]{Quantifying U.S. GATT Trade Liberalization}}
```
Comment (MatthewRGonzalez) [inline data-paper/presentation.rmd:11]: Changed the hspace in order to deal with the title in the bottom bar (somehow the able figures moved the text around beforehand).

File: data-paper/presentation.rmd [hunk 2]
```diff
@@ -8,11 +8,12 @@ output:
...
   - \AtBeginDocument{\author[Buzard (kbuzard@syr.edu), Jestrab and Xiong]{Kristy Buzard, Syracuse University, kbuzard@syr.edu\\ Ross Jestrab, Syracuse University\\ Zeyuan (Victor) Xiong, Syracuse University}}
   - \usecolortheme{beaver}
   - \AtBeginSection{}
   - \AtBeginSubsection{}
+  - \usepackage{booktabs}
```
Comment (MatthewRGonzalez) [inline data-paper/presentation.rmd:16]: Necessary in order to include kable tables in beamer

File: data-paper/presentation.rmd [hunk 3]
```diff
@@ -108,17 +109,21 @@ num_nego <- c(0,nr-nrow(sameSH_BG),nr-nrow(sameBG_Ge),nr-nrow(sameGe_An),nr-nrow
...
+<!--   geom_point(data = plot1_data, aes(x=year, y= num_nego), color = "red") + -->
+<!--   annotate("text", label = "No. of items reduced since previous round", x = 1952, y = 50, size = 5, colour = "red") + -->
+<!--   labs(x="Year", y= "Number of Items") + -->
+<!--   theme(plot.title = element_text(hjust = 0.5)) + -->
+<!--   scale_x_continuous(breaks = seq(1930, 1965, by = 5)) + -->
+<!--   scale_y_continuous(breaks = seq(0, 3000, by = 500)) -->
+<!-- ``` -->
+
+```{r, echo=FALSE, out.width="75%", fig.cap=""}
+knitr::include_graphics("data-paper_files/figure-latex/ibi-1.pdf")
```
Comment (MatthewRGonzalez) [inline data-paper/presentation.rmd:126]: The directory might have to be changed here "data-paper/files/etc..."

File: data-paper/presentation.rmd (hunk 1 above)
Comment (kbuzard) [reply data-paper/presentation.rmd:11]: I had this experience too. It's baffling, but it's reality.

**Comment by kbuzard** [comment]
@MatthewRGonzalez This looks GREAT! I merged the pull request and ran it and it all looks better than I dreamed possible.

I'm going to keep working on the "talk-y" slides while you do as much of the rest as you can--just let me know if you need to stop short of completing the rest (some of it is not totally necessary anyway).

There's just one small improvement I can see to the code: given that you're just inserting the item-level graph directly, the whole big code chunk before it can be deleted--it's not used anywhere else in the code, it's only purpose was to make that table.

**Comment by MatthewRGonzalez** [comment]
> There's just one small improvement I can see to the code: given that you're just inserting the item-level graph directly, the whole big code chunk before it can be deleted--it's not used anywhere else in the code, it's only purpose was to make that table.

No problem, I’ll address this in my next PR. Taking a break now but I’ll be back on in an hour to take a look at the remaining tasks

**Pull request merged**